package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apollo-upload-client",
-  "version": "6.0.2",
+  "version": "6.0.3",
```

Comment:
Best to undo this; I'll update the changelog and version in a few places myself prior to a release.